src/Microphysics2M.jl

```diff
@@ -71,18 +97,29 @@ function conv_q_liq_to_q_rai_B1994(
     q_liq::FT,
     ρ::FT;
     N_d::FT = 1e8,
+    smooth_transition::Bool = false,
 ) where {FT <: Real}
 
-    q_liq = max(0.0, q_liq)
+    q_liq = max(0, q_liq)
 
     C::FT = CMP.C_acnv_B1994(param_set)
     a::FT = CMP.a_acnv_B1994(param_set)
     b::FT = CMP.b_acnv_B1994(param_set)
     c::FT = CMP.c_acnv_B1994(param_set)
     N_0::FT = CMP.N_0_B1994(param_set)
-    d::FT =
-        N_d >= N_0 ? CMP.d_low_acnv_B1994(param_set) :
-        CMP.d_high_acnv_B1994(param_set)
+    d::FT = 0.0
```

Comment:
Do we have to declare d here?

Reply:
cpp experience effect :)

Reply:
Maybe I did it because you should declare type of a variable only once and I wanted to declare before if statement. I don't know which option is clearer!

Reply:
I didn't know that. Lets just go with what was here before then